### PR TITLE
Added PHP support

### PR DESCRIPTION
**Pull request by stefanak-michal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@observablehq/framework",
   "license": "ISC",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (mbostock) [inline package.json:4]: Please revert this change. We’ll bump the version when we prepare a release. Thank you.

**Comment by Fil** [comment]
wrapped under https://github.com/observablehq/framework/pull/935 ; let us know what you think of the proposal and questions it raises.
